### ISSUE TITLE
Cool Graphical printing option for sauced show

### ISSUE THREAD

**Issue by mdsauce** [maxdobeck/sauced#47]
`sauced show` just outputs raw JSON, which should not be changed!  Raw JSON can be useful if you're piping output.

However `sauced show --pretty` or `sauced show -p` should output something colored (ideally) using the logrus package https://github.com/sirupsen/logrus.

Or something kind of custom using something really cool like https://github.com/gizak/termui.  Maybe a `sauced show --vaporwave` or something.

Or https://github.com/jroimartin/gocui

Basically make cool display options a thing.
